samples/disk_resize.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+"""
+Written by Sahil Gandhi
+Github: https://github.com/sahilmgandhi
+
+This code is released under the terms of the Apache 2
+http://www.apache.org/licenses/LICENSE-2.0.html
+
+Resize a disk that is attached to a VM to the desired size.
+"""
+
+import atexit
+import requests
+import argparse
+
+from tools import cli
+from pyVmomi import vim
+
+from pyVim.connect import SmartConnectNoSSL, Disconnect
+from tools import tasks
+
+__author__ = 'sahilmgandhi'
+
+
+def modify_disk(si, vm_obj, disk_number, size):
+
+    disk_label = 'Hard disk ' + str(disk_number)
+    vm_disk = None
+
+    # Search for the vm disk
+    for dev in vm_obj.config.hardware.device:
+        if isinstance(dev, vim.vm.device.VirtualDisk) \
+                and dev.deviceInfo.label == disk_label:
+            vm_disk = dev
+    if not vm_disk:
+        raise RuntimeError(
+            'Virtual {} could not be found.'.format(disk_label))
+
+    # Configure the vm disk with the appropriate size
+    vm_disk_spec = vim.vm.device.VirtualDeviceSpec()
+    vm_disk_spec.operation = vim.vm.device.VirtualDeviceSpec.Operation.edit
+    vm_disk_spec.device = vm_disk
+    vm_disk_spec.device.capacityInKB = size * 1024 * 1024
+
+    # Actually go and change the vm disk
+    dev_changes = []
+    dev_changes.append(vm_disk_spec)
+    spec = vim.vm.ConfigSpec()
+    spec.deviceChange = dev_changes
+    task = vm_obj.ReconfigVM_Task(spec=spec)
+    tasks.wait_for_tasks(si, [task])
+    return True
+
+
+def get_args():
+
+    # Parse through the arguments. Style and format taken from the other
+    # samples in the samples directory
+    parser = argparse.ArgumentParser()
```

Comment:
Most of these arguments are already handled in the cli.py helper class. Can you use it instead.